tests/test_model.py

```diff
@@ -1833,8 +1823,12 @@ def test_open_access_loan(self):
         eq_(loan, loan2)
         eq_(False, was_new)
 
+
     def test_work(self):
         """Test the attribute that finds the Work for a Loan or Hold."""
+        # TODO:  determine if using pool.presentation_edition is correct for lending.
+        # Because, while the old pool.edition was tied to something you can lend, the presentation_edition 
+        # is not representing a lendable object, really.  
```

Comment:
I don't understand this comment. The `LicensePool` is always something you can lend, and its `.edition` (old) or `.presentation_edition` (new) associated with it never represented anything lendable.